src/changelog.ts

```diff
@@ -264,28 +354,42 @@ export default class Changelog {
 
   #formatter: Formatter;
 
+  readonly #versionBeforePkgRename: string | undefined;
+
+  readonly #tagPrefixBeforePkgRename: string | undefined;
+
   /**
    * Construct an empty changelog.
    *
    * @param options - Changelog options.
    * @param options.repoUrl - The GitHub repository URL for the current project.
    * @param options.tagPrefix - The prefix used in tags before the version number.
    * @param options.formatter - A function that formats the changelog string.
+   * @param options.versionBeforePkgRename - A version string of the package before being renamed.
+   * An optional, which is required only in case of package renamed.
+   * @param options.tagPrefixBeforePkgRename - A tag prefix string of the package before being renamed.
+   * An optional, which is required only in case of package renamed.
```

Comment:
Applying JSDoc syntax for optional params.
```suggestion
   * @param [options.tagPrefix] - The prefix used in tags before the version number.
   * @param [options.formatter] - A function that formats the changelog string.
   * @param [options.versionBeforePkgRename] - A version string of the package before being renamed.
   * An optional, which is required only in case of package renamed.
   * @param [options.tagPrefixBeforePkgRename] - A tag prefix string of the package before being renamed.
   * An optional, which is required only in case of package renamed.
```